resources/public/api/conf/2.0/examples/foreign_property_annual_submission_create_and_amend/request.json

```diff
@@ -0,0 +1,48 @@
+{
+  "foreignFhlEea": {
+    "adjustments": {
+      "privateUseAdjustment": 34343.45,
+      "balancingCharge": 53543.23,
+      "periodOfGraceAdjustment": true
+    },
+    "allowances": {
+      "annualInvestmentAllowance": 3434.23,
+      "otherCapitalAllowance": 1343.34,
+      "electricChargePointAllowance": 6565.45,
+      "zeroEmissionsCarAllowance": 3456.34,
+      "propertyIncomeAllowance": 4004.49
```

Comment:
This would cause both RULE_BOTH_ALLOWANCES_SUPPLIED and RULE_PROPERTY_INCOME_ALLOWANCE errors.  Need to take propertyIncomeAllowance out.